src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -3812,6 +3871,92 @@ bool emberAfDoorLockClusterClearHolidayScheduleCallback(
     return true;
 }
 
+bool emberAfDoorLockClusterSetAliroReaderConfigCallback(CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
+                                                        const Commands::SetAliroReaderConfig::DecodableType & commandData)
+{
+    DoorLockServer::Instance().setAliroReaderConfigCommandHandler(commandObj, commandPath, commandData.signingKey,
+                                                                  commandData.verificationKey, commandData.groupIdentifier,
+                                                                  commandData.groupResolvingKey);
+    return true;
+}
+
+bool emberAfDoorLockClusterClearAliroReaderConfigCallback(CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
+                                                          const Commands::ClearAliroReaderConfig::DecodableType & commandData)
+{
+    DoorLockServer::Instance().clearAliroReaderConfigCommandHandler(commandObj, commandPath);
+    return true;
+}
+
+void DoorLockServer::setAliroReaderConfigCommandHandler(CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
+                                                        const ByteSpan & signingKey, const ByteSpan & verificationKey,
+                                                        const ByteSpan & groupIdentifier,
+                                                        const Optional<ByteSpan> & groupResolvingKey)
```

Comment:
These various ByteSpans need length checks to make sure they are the right sizes, right?  Those need to happen around the same place as the groupResolvingKey presence check, before the readerVerificationKey check.

Reply:
done.